test/functional/decorators/foreign-key/wrong city.ts

```diff
@@ -0,0 +1,21 @@
+import {
+    Column,
+    Entity,
+    ForeignKey,
+    PrimaryColumn,
+    Unique,
+} from "../../../../src"
+
+@Entity("wrong_cities")
+@Unique(["id", "countryCode"])
+export class WrongCity {
```

Comment:
This file appears to have a space in its name `wrong city.ts` - should use kebab-case